setup.py

```diff
@@ -5,21 +5,13 @@
 
 setuptools.setup(
     name="jf_agent",
-    version="0.0.21",
+    version="0.0.20",
```

Comment:
You meaning to decrement the version?

Reply:
weirdly, yes: we apparently never actually released a .20, so we bumped it to .21 before unnecessarily.